R/SchemaGenerator.R

```diff
@@ -14,11 +14,11 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-.writeFieldDefinition <- function(field) {
-  field <- as.list(field)
-  str <- paste("\t", field$fieldName, toupper(field$dataType))
+.writeColumnDefinition <- function(column) {
+  column <- as.list(column)
+  str <- paste("\t", column$columnName, toupper(column$dataType))
 
-  if (field$primaryKey == "yes") {
+  if (column$primaryKey == "yes") {
```

Comment:
if we change this to:

`if (tolower(column$primaryKey) == "yes")`

We should avoid issues with casing in the csv file. (Though I note this sort of thing would not be a problem if we convert to using json structures in the future).